File: 1672-Richest-Customer-Wealth/1672.py
class Solution:
    def maximumWealth(self, accounts: List[List[int]]) -> int:
        num_rows = len(accounts)
        num_columns = len(accounts[0]) 
        richest = 0
        for i in range (num_rows):
            temp = 0
            for j in range (num_columns):
                temp += accounts[i][j]
            if (temp > richest):
                richest = temp
        return richest

File: 1672-Richest-Customer-Wealth/1672.cpp
class Solution {
public:
    int maximumWealth(vector<vector<int>>& accounts) {
        int num_rows = accounts.size();
        int num_columns = accounts[0].size();
        int richest = 0;
        for (int i=0;i<num_rows;++i){
            int tempSum = 0;
            for (int j=0;j<num_columns;++j){
                tempSum+= accounts[i][j];
            }
            if (tempSum>richest){
                richest = tempSum;
            }
        }
        return richest;
    }
};
